src/types.ts

```
@@ -56,6 +56,7 @@ export type OperationObject = OpenAPIV3.Document | OpenAPIV3_1.OperationObject
 
 export type PlaygroundSecurityScheme = OpenAPIV3.HttpSecurityScheme & OpenAPIV3.ApiKeySecurityScheme & OpenAPIV3.OAuth2SecurityScheme & OpenAPIV3.OpenIdSecurityScheme & {
   playgroundValue: RemovableRef<any>
+      // | any
```

ESLint

[error] Expected indentation of 2 spaces but found 6.

(style/indent)